### ISSUE TITLE
PHP Fatal error:  Allowed memory size exhausted

### ISSUE THREAD

**Issue by Marty** [mondediefr/docker-flarum#62]
When trying to run
```
docker exec -ti flarum_flarum_1 extension require fof/links
```
I get the following error
``` 
PHP Fatal error:  Allowed memory size of 1610612736 bytes exhausted (tried to allocate 4096 bytes) in phar:///usr/local/bin/composer/src/Composer/DependencyResolver/RuleWatchGraph.php on line 52
```
I already have `PHP_MEMORY_LIMIT: 1G` set in my `docker-compose.yml` which gets applied correctly
```
/ # cat /etc/php7/php-fpm.conf | grep memory_limit
php_admin_value[memory_limit] = 1G
```
I tried upgrading composer to the latest 1.x which did not fix it and 2.x is not compatible with flarum (?).
The host still has 11gb available.

Any ideas?

**Comment by Hydrog3n** [comment]
Hello, @Marty thank you for your issue. 
It's saying it's trying to use 1.6GB of ram so can you change PHP_MEMORY_LIMIT to 2G or try -1 (unlimited)?

**Comment by Marty** [comment]
Yes, I already had done that too (-1), but it persisted. It seemed that my docker container somehow had a memory limitation, at least according to top. Now - a few days later - it works. Also top show more free memory ... I do not know what caused this behaviour.

**Comment by Hydrog3n** [comment]
Ok Top if it's work now. Maybe a previous mem limit in docker-compose.yml

**Comment by Marty** [comment]
@Hydrog3n I ran into this again today, this time while starting the container.
After further testing I added the following to my `docker-compose.yml` and the issue disappeared:
```
services:
  flarum:
    environment:
      COMPOSER_MEMORY_LIMIT: -1
```
I "think" the composer uses the `php.ini` rather than the `php-fpm.conf` (see [here](https://getcomposer.org/doc/articles/troubleshooting.md#memory-limit-errors)), but setting the env variable overrides this.

**Comment by Magicalex** [comment]
Can you retry with the lastest tag?

**Comment by Magicalex** [comment]
@Marty fixed in this commit 914b3fcc4b43c5b3d5
